docs/README.md

```diff
@@ -0,0 +1,152 @@
+<h1 align = "center">Image Classification App Boilerplate</h1>
+
+Have you been puzzled by tons of videos, blogs and other resources on the internet and don't know where and how to deploy your AI models?
+<p align = "center"><img src = "https://media.giphy.com/media/8mndEBLsg9Whg2Sduv/giphy.gif" width = 30%></p>
+
```

Comment:
Do not just change the Markdown file location for mkdocs. You are supposed to create a whole new mkdocs project inside a directory mkdocs. If this was to be done, then there is no point of having the issue raised